src/test/adapter-factory/auth-flow.ts

```
@@ -0,0 +1,179 @@
+		const user = await generate("user");
+		const start = Date.now();
+		const result = await auth.api.signUpEmail({
+			body: {
+				email: user.email,
+				password: crypto.randomUUID(),
+				name: user.name,
+				image: user.image || "",
+			},
+		});
+		const end = Date.now();
+		console.log(`signUpEmail took ${end - start}ms (without hashing)`);
```

ESLint

[error] Unexpected console statement.

(no-console)